app/views/npc_pdfs/bland_show.html.erb

```diff
@@ -0,0 +1,177 @@
+        <li>Charisma: <%= @npc.core_modifiers_cha_mod %></li>
+    <p>Saving Throws:</p>
+      <li>Strength: <%= @npc.saving_throws_str %></li>
+      <li>Dexterity: <%= @npc.saving_throws_dex %></li>
+      <li>Constitution: <%= @npc.saving_throws_con %></li>
+      <li>Intelligence: <%= @npc.saving_throws_int %></li>
+      <li>Wisdom: <%= @npc.saving_throws_wis %></li>
+      <li>Charisma: <%= @npc.saving_throws_cha %></li>
+    <p>Skills:</p>
+      <li>Acrobatics: <%= @npc.skills_acrobatics %></li>
+      <li>Animal Handling: <%= @npc.skills_animal_handling %></li>
+      <li>Arcana: <%= @npc.skills_arcana %></li>
+      <li>Athletics: <%= @npc.skills_athletics %></li>
+      <li>Deception: <%= @npc.skills_deception %></li>
+      <li>History: <%= @npc.skills_history %></li>
+      <li>Insight: <%= @npc.skills_insight %></li>
+      <li>Intimidation: <%= @npc.skills_intimidation %></li>
+      <li>Investigation: <%= @npc.skills_investigation %></li>
+      <li>Medicine: <%= @npc.skills_medicine %></li>
+      <li>Nature: <%= @npc.skills_nature %></li>
+      <li>Perception: <%= @npc.skills_perception %></li>
+      <li>Performance: <%= @npc.skills_performance %></li>
+      <li>Persuasion: <%= @npc.skills_persuasion %></li>
+      <li>Religion: <%= @npc.skills_religion %></li>
+      <li>Sleight of Hand: <%= @npc.skills_sleight_of_hand %></li>
+      <li>Stealth: <%= @npc.skills_stealth %></li>
+      <li>Survival: <%= @npc.skills_survival %></li>
+
+      <h3>Spells:</h3>
+        <p>Spell_slots: <%= @npc.spell_slots %></p>
+        <p>Current Spells:</p>
+          <% if @npc.spells.class == String %>
+            <%= @npc.spells %>
+          <% else %>
+            <% @npc.spells.each do |spell| %>
+              <li>Name: <%= spell[:name] %></li>
+              <li>Description: <%= spell[:description] %></li>
+              <li>Range: <%= spell[:range] %></li>
+              <li>Components: <%= spell[:components] %></li>
+              <li>Duration: <%= spell[:duration] %></li>
+              <li>Casting Time: <%= spell[:casting_time] %></li>
+              <li>Level: <%= spell[:level] %></li>
+              <li>School: <%= spell[:school] %></li>
+              <li>Classes: <%= spell[:classes] %></li>
+              <li>Attack Type: <%= spell[:attack_type] %></li>
+              <li>Concentration: <%= spell[:concentration] %></li>
+              <li>Damage Type: <%= spell[:damage_type] %></li>
+              <li>Damage at Character Level: <%= spell[:damage_at_character_level] %></li>
+              <li>Damage At Slot Level: <%= spell[:damage_at_slot_level] %></li>
+              <li>Higher_level: <%= spell[:higher_level] %></li>
+              <li>Material: <%= spell[:material] %></li>
+              <li>Ritual: <%= spell[:ritual] %></li>
+              <li>Subclasses: <%= spell[:subclasses] %></li>
+              -----------
+            <% end %>
+          <% end %>
+</div>
```

Comment:
This is basically just the usual view, but turned into a PDF.  This could be used as a basis to build around if we wanted to just make our own PDF template instead of the copy/pasted one that is the current `show`